src/utils/request-queue.js

```diff
@@ -15,8 +15,9 @@ class RequestQueue {
     this.requestPending = false;
     this.pageUnloading = false;
 
-    // eslint-disable-next-line no-unneeded-ternary
-    this.sendTrackingEvents = (options && options.sendTrackingEvents === false) ? false : true;
+    this.sendTrackingEvents = (options && options.sendTrackingEvents === false)
```

Comment:
Would `this.sendTrackingEvents = !!options.sendTrackingEvents` have the same effect?

Reply:
Jimmy and I chatted about this a while when we put this into place. Essentially, it makes it explicit that you must define a boolean. Otherwise you could do things like `sendTrackingEvents: "no"` and it would be truthy.